core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml

```diff
@@ -1,8 +1,8 @@
 templates:
   main: !template
-    reportTemplate: 'dummy.jxml'
+    reportTemplate: 'dummy.jrxml'
```

Comment:
why to change this one as jxml is used everywhere else?

Reply:
`jxml` is used nowhere (expect here)!

Reply:
grep -r 'dummy.jxml' ./ | wc = 106 occurences

Reply:
Oups, fixed :-)